### PR TITLE
Make small corrections to Programming chapter

### PR DESCRIPTION
**Pull request by StevenMaude**
Small editorial fixes.

### CODE REVIEW

File: text/en/chapters/programming-languages.md [hunk 1]
```diff
@@ -510,7 +510,7 @@ The diagram below shows the difference between what happens in an interpreter an
...
+The languages we have discussed in this chapter are ones that you are likely to come across in introductory programming, but there are some completely different styles of languages that have very important applications. There is an approach to programming called [Functional programming](http://en.wikipedia.org/wiki/Functional_programming) where all operations are formulated as mathematical functions. Common languages that use functional techniques include Lisp, Scheme, Haskell, Clojure and F#; even some conventional languages (such as Python) include ideas from functional programming. A pure functional programming style eliminates a problem called *side effects*, and without this problem it can be easier to make sure a program does exactly what it is intended to do. Another important type of programming is [logic programming](http://en.wikipedia.org/wiki/Logic_programming), where a program can be thought of as a set of rules stating what it should do, rather than instructions on how to do it. The most well-known logic programming language is Prolog.
```
Comment (StevenMaude) [inline text/en/chapters/programming-languages.md]: The change here is just "Haskel" > "Haskell"; GitHub doesn't seem to highlight the word change here.

**Comment by StevenMaude** [comment]
> We will start with looking at some other programming languages that programmers use to give instructions to computers, then we will look at low level languages and how computers actually carry out the instructions in them...

This is also incorrect: the MIPS section appears before the other languages section. Would have fixed this, but weren't sure if you'd rather correct this sentence, or reorganise the section so they agree with this statement.

**Comment by JackMorganNZ** [comment]
@StevenMaude I'll ask an author on the intended order of sections, then merge your request in.

**Comment by Heidirose** [comment]
That chapter is going to be undergoing a significant refactoring in the
near future

On Mon, Mar 7, 2016 at 3:46 PM, Jack Morgan <notifications@github.com>
wrote:

> @StevenMaude <https://github.com/StevenMaude> I'll ask an author on the
> intended order of sections, then merge your request in.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/uccser/cs-field-guide/pull/248#issuecomment-193065797>
> .
>



-- 
Heidi Newton
Computer Science Coach
Phone: 0221009239

**Comment by JackMorganNZ** [comment]
@StevenMaude Since @Heidirose is rewriting the whole chapter in the future, I'll fix the wording of the introduction to reflect the current section ordering. Heidi can reorganise it later.

Thanks for your hard work.

**Comment by StevenMaude** [comment]
No problem! It's a great resource.

**Pull request merged**